### PR TITLE
add bottom nav, link all screens

### PR DESCRIPTION
**Pull request by ybxiang05**
- link all screens, including modals
- add bottom nav bar

### CODE REVIEW

File: client/js/App.js [hunk 1]
```diff
@@ -18,21 +18,15 @@ import {
...
+import RootStack from './navigation';
 
 const App = () => {
   return (
     <NavigationContainer>
-      <Text style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
+      {/* <Text style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
         Feed Me
-      </Text>
+      </Text> */}
```
Comment (siddharthparmar7) [inline client/js/App.js]: remove commented code if not needed.

File: client/js/components/CustomText/CustomText.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from './styles';
+import {Text} from 'react-native';
+
+export default class CustomText extends React.Component {
```
Comment (siddharthparmar7) [inline client/js/components/CustomText/CustomText.js]: This should be stateless component.

File: client/js/navigation/navigation.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+const NavScreens = props => {
+  return (
+    <Nav.Navigator
+      tabBarOptions={{
+        activeTintColor: 'green',
+        inactiveTintColor: '#CACACA',
+        labelStyle: {
+          fontSize: 11,
+          // fontFamily: 'Montserrat',
```
Comment (siddharthparmar7) [inline client/js/navigation/navigation.js:83]: remove commented code if not needed.

File: client/js/screens/Ethics/Ethics.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import {Text} from 'react-native';
+// import CText from '../../components/CustomText';
```
Comment (siddharthparmar7) [inline client/js/screens/Ethics/Ethics.js:3]: same as above.

File: client/js/screens/Guide/Guide.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Text, TouchableHighlight} from 'react-native';
+// import CText from '../../components/CustomText';
```
Comment (siddharthparmar7) [inline client/js/screens/Guide/Guide.js:3]: same as above

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the requested changes. Thank you.

File: client/js/navigation/navigation.js (hunk 3 above)
Comment (ybxiang05) [reply client/js/navigation/navigation.js:83]: Still figuring out how to add icons and fonts, needed to push so team can have basic nav structure to work with. Commented out code will be needed soon.

File: client/js/screens/Ethics/Ethics.js (hunk 4 above)
Comment (ybxiang05) [reply client/js/screens/Ethics/Ethics.js:3]: Part of figuring out custom text, will be needed later

File: client/js/screens/Guide/Guide.js (hunk 5 above)
Comment (ybxiang05) [reply client/js/screens/Guide/Guide.js:3]: Will be needed later

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**